travis/Dockerfile-travis-watch

```diff
@@ -1,4 +1,4 @@
-FROM mitodl/bootcamp_watch_travis
+FROM mitodl/bootcamp_watch_travis_next
```

Comment:
@noisecapella What's the `next` for?

Reply:
We use these uploaded images to skip some of the installation during travis tests

Reply:
The `next` part lets us have the current image and the future image uploaded at the same time, so this PR can coexist with other PRs